src/app.scss

```diff
@@ -73,6 +73,14 @@ body {
 	--surface-nine: hsl(calc(var(--hue, 206) + 24), 9.5%, 17.5%);
 
 	--red-one: hsl(333, 84%, 62%);
+	--red-two: hsl(357.14, 100%, 20.59%);
+	--red-three: hsl(355.92, 100%, 28.82%);
+	--red-four: hsl(5.85, 100%, 91.96%);
+
+	--yellow-one: hsl(0, 0%, 0%);
+	--yellow-two: hsl(49.57, 100%, 4.51%);
+	--yellow-three: hsl(50.3, 39.13%, 50.39%);
+	--yellow-four: hsl(49.64, 85.94%, 74.9%);
```

Comment:
You should use colors similar to:

Red: hsl(357, 74%, 60%);
Yellow: hsl(59, 100%, 72%);

Reply:
red is the one the m3 theme creator outputted using the default color of the website

Reply:
It's way too darky a lighter color like proposed is necessary

Reply:
resolved in 79df7ad